coalib/parsing/LineParser.py

```diff
@@ -82,6 +82,15 @@ def _parse(self, line):
             line,
             self.comment_separators)
         comment = unescape(comment)
+
+        if '#' in comment:
+            comment_index = comment.index('#')
+            if comment_index + 1 < len(comment) or comment_index - 1 > 0:
+                if comment[comment.index('#') + 1] != ' '\
+                   or comment[comment.index('#') - 1] != ' ':
+                    logging.warning('The comment does not have whitespace' +
```

Comment:
Imagine getting a random "The comment does not have whitespace" out of nowhere!

Since we have no access to the line number, we should at least print the line so that the user knows where to look.